### ISSUE TITLE
Fixes for File upload and mailer

### ISSUE THREAD

**Issue by adamshelley31** [Pro-Enviro/IDNI#143]
1) FIle upload to reset on change company to prevent files carrying over to another company.
2) Update subject in emailer to includes company line
3) Enable sending of multiple files to show in the email

**Comment by adamshelley31** [comment]
Added Mila in emailer for testing. Adam change back when merging: 
**from:**
to: ['it@proenviro.co.uk', 'data@proenviro.co.uk','richard.pelan@investni.com'],
**to**:
to: ['mila.lozanova@proenviro.co.uk'],

**Comment by mila-proenviro** [comment]
1) FIle upload to reset on change company to prevent files carrying over to another company.
~~2) Update subject in emailer to includes company line~~
~~3) Enable sending of multiple files to show in the email~~

**Comment by mila-proenviro** [comment]
Uploading HH Data for Capital test 2 and capital test 3
1. After uploding data for Capital Test 2, I started to upload data for Capital Test 3 and the "HHD" table wasn't cleared out from the previous company. So the information such as MPAN, date column and data start point was already filled.
<img width="1536" alt="Screenshot 2024-07-08 at 09 32 40" src="https://github.com/Pro-Enviro/IDNI/assets/144015071/b14a0491-3eff-472c-8e99-6a4ad87946c1">

2. After uploading the hhd data for Capital test 3, the company has 2 mpans - the mpan that I imported and the mpan from the previous company.

<img width="1536" alt="Screenshot 2024-07-08 at 09 33 18" src="https://github.com/Pro-Enviro/IDNI/assets/144015071/4f1c00ad-bc8a-475f-b6c5-84297b13cff0">
